.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,67 +1,72 @@
-[Short description explaining the high-level reason for the pull request]
+_[Enter an explanation of what the pull request does and why.]_
 
-## Additions
 
--
-
-## Removals
-
--
+---
 
-## Changes
+_[Feel free to delete any checkboxes that are not applicable to this PR.]_
 
--
 
-## Testing
+## How to test this PR
 
 1.
 
+
 ## Screenshots
 
 
-## Notes
+## Notes and todos
 
 -
 
-## Todos
-
--
 
 ## Checklist
 
+_[Feel free to delete any checkboxes that are not applicable to this PR.]_
+
 - [ ] PR has an informative and human-readable title
+  - PR titles are used to generate the change log in [releases](../../releases),
+    so good ones make that easier to scan.
+  - Consider prefixing with the feature/section you're addressing,
+    e.g., "Mega Menu: fix layout bug" or "Paying for College: Update content on Repay tool"
 - [ ] Changes are limited to a single goal (no scope creep)
-- [ ] Code can be automatically merged (no conflicts)
 - [ ] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
-- [ ] Passes all existing automated tests
-- [ ] Any _change_ in functionality is tested
-- [ ] New functions are documented (with a description, list of inputs, and expected output)
-- [ ] Placeholder code is flagged / future todos are captured in comments
-- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
+- [ ] Future todos are captured in comments and/or tickets
 - [ ] Project documentation has been updated
-- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:
+  - Potential places for documentation:
+    - [This repo’s docs](https://cfpb.github.io/cfgov-refresh/) (edit the files in the `/docs` folder) –
+      for basic, close-to-the-code documentation on working with this repo
+    - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
+    - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
 
-## Testing checklist
+### Front-end testing
 
-### Browsers
+_[When new or significantly modified front-end functionality is present,
+  the following things should be tested. Feel free to delete this section
+  if not applicable to this PR.]_
 
-- [ ] Chrome on desktop
+#### Browser testing
+
+Visually tested in the following supported browsers:
 - [ ] Firefox
-- [ ] Safari on macOS
-- [ ] Edge
-- [ ] Internet Explorer 9, 10, and 11
+- [ ] Chrome
+- [ ] Safari
+- [ ] Edge 18
```

Comment:
Specifying 18 here as the last major version of Edge prior to its Chromium release. Haven't added a Chromium version explicitly, because it's not the default browser installed with Windows yet.

Reply:
Does Edge auto update? If so, should we only check it in the latest version?

Reply:
That's what I was getting at. It's not auto-updating from 18, the last Trident release, to the new Chromium series.